disasm.py

```diff
@@ -148,76 +148,106 @@ def addComments(script, actionLines):
                 isFloatMask = 0xF000000
                 extractMSBMask = 0xFF000000
                 extractDecimalIDMask = 0xFFFFFF
-                if parameter & isBitOrBasicMask: # if basic or bit
+                if isinstance(parameter, int) and parameter & isBitOrBasicMask: # if basic or bit
                     mostSignificantBytes = (parameter & extractMSBMask) >> (4 * 6)
                     decimalID = parameter & extractDecimalIDMask
-                    if mostSignificantBytes == int('10',16):
+                    if mostSignificantBytes == 0x10:
                         script[i].debugString = "LA-Basic {}".format(decimalID)
-                    elif mostSignificantBytes == int('11',16):
+                    elif mostSignificantBytes == 0x11:
                         script[i].debugString = "RA-Basic {}".format(decimalID)
-                    elif mostSignificantBytes == int('12',16):
+                    elif mostSignificantBytes == 0x12:
                         script[i].debugString = "fighter_param_common-Basic {}".format(decimalID)
-                    elif mostSignificantBytes == int('13',16):
+                    elif mostSignificantBytes == 0x13:
                         script[i].debugString = "fighter_param-Basic {}".format(decimalID)
-                    elif mostSignificantBytes == int('20',16):
+                    elif mostSignificantBytes == 0x20:
                         script[i].debugString = "LA-Bit {}".format(decimalID)
-                    elif mostSignificantBytes == int('21',16):
+                    elif mostSignificantBytes == 0x21:
                         script[i].debugString = "RA-Bit {}".format(decimalID)
-                elif parameter & isFloatMask: # if float
+                    elif mostSignificantBytes == 0x1e:
+                        script[i].debugString = "Action Status-Type1? {}".format(decimalID)
```

Comment:
Small nitpick but could you use % based formatting instead of .format ? just a personal preference and would like consistency throughout the repository